app/assets/javascripts/utils/article_utils.js

```diff
@@ -30,4 +30,39 @@ export default class ArticleUtils {
     };
     return mapping[namespace];
   }
+
+  static NamespaceTitleFromId = {
```

Comment:
If this mapping is duplicated in Ruby, both places should include a comment pointing to the other one so that we are less likely to update one without updating the other.